templates/repo/settings/secrets.tmpl

```diff
@@ -0,0 +1,60 @@
+<div class="ui container">
+		<h4 class="ui top attached header">
+			{{.locale.Tr "secrets.secrets"}}
+			<div class="ui right">
+				<div class="ui primary tiny show-panel button" data-panel="#add-secret-panel">{{.locale.Tr "secrets.creation"}}</div>
+			</div>
+		</h4>
+		<div class="ui attached segment">
+			<div class="{{if not .HasError}}hide {{end}}mb-4" id="add-secret-panel">
+				<form class="ui form" action="{{.Link}}?act=secret" method="post">
+					{{.CsrfTokenHtml}}
+					<div class="field">
+						{{.locale.Tr "secrets.description"}}
+					</div>
+					<div class="field {{if .Err_Title}}error{{end}}">
+						<label for="title">{{.locale.Tr "secrets.name"}}</label>
+						<input id="ssh-key-title" name="title" value="{{.title}}" autofocus required pattern="^[a-zA-Z_][a-zA-Z0-9_.-]*$">
+					</div>
+					<div class="field {{if .Err_Content}}error{{end}}">
+						<label for="content">{{.locale.Tr "secrets.value"}}</label>
+						<textarea id="ssh-key-content" name="content" required placeholder="{{.locale.Tr "secrets.creation.value_placeholder"}}">{{.content}}</textarea>
+					</div>
+					<button class="ui green button">
+						{{.locale.Tr "secrets.creation"}}
+					</button>
+					<button class="ui hide-panel button" data-panel="#add-secret-panel">
+						{{.locale.Tr "cancel"}}
+					</button>
+				</form>
+			</div>
+			{{if .Secrets}}
+				<div class="ui key list">
+					{{range .Secrets}}
+						<div class="item">
+							<div class="right floated content">
+								<button class="ui red tiny button delete-button" data-url="{{$.Link}}/delete?act=secret" data-id="{{.ID}}">
+									{{$.locale.Tr "settings.delete_key"}}
```

Comment:
GitHub warns before deleting a secret in any case. I think we should do that too.

Reply:
I think we did that:

<img width="1215" alt="image" src="https://user-images.githubusercontent.com/9418365/208397181-5f68bfa1-53cc-4db6-9104-6fbfc85a8bab.png">

Reply:
@wolfogre Yes, in the org settings.
I commented on the repo secrets explicitly, because as far as I can see, this dialogue is currently missing here.

EDIT: Okay, I've now been able to test it, it is there because of the deploy keys. However, as it is hardcoded on the deploy keys, it looks wrong for secrets: 
![image](https://user-images.githubusercontent.com/51889757/208423945-a6bc1d11-89c2-4f77-940d-4d18dd77d236.png)